src/rpc/misc.cpp

```diff
@@ -367,6 +368,7 @@ UniValue validateaddress(const UniValue& params, bool fHelp)
             "  \"address\" : \"pivxaddress\", (string) The pivx address validated\n"
             "  \"scriptPubKey\" : \"hex\",       (string) The hex encoded scriptPubKey generated by the address\n"
             "  \"ismine\" : true|false,          (boolean) If the address is yours or not\n"
+            "  \"iscoldstaking\" : true|false,   (boolean) If the address is a PIVX cold staking address\n"
```

Comment:
Came to look here to get an idea of the standard for rpc outputs; and I see some improvement that could be made:
```
Result:
{
  "isvalid" : true|false,         (boolean) If the address is valid or not. If not, this is the only property returned.
  "address" : "pivxaddress", (string) The pivx address validated
  "scriptPubKey" : "hex",       (string) The hex encoded scriptPubKey generated by the address
  "ismine" : true|false,          (boolean) If the address is yours or not
  "iscoldstaking" : true|false,   (boolean) If the address is a PIVX cold staking address\n"
  "iswatchonly" : true|false,   (boolean) If the address is watchonly
  "isscript" : true|false,        (boolean) If the key is a script
  "hex" : "hex",                (string, optional) The redeemscript for the P2SH address
  "pubkey" : "publickeyhex",    (string) The hex value of the raw public key
  "iscompressed" : true|false,    (boolean) If the address is compressed
  "account" : "account"         (string) The account associated with the address, "" is the default account
}
```
Could use to be better aligned
```
  "isvalid" : true|false,       (boolean) If the address is valid or not. If not, this is the only property returned.
  "address" : "pivxaddress",    (string)  The pivx address validated
  "scriptPubKey" : "hex",       (string)  The hex encoded scriptPubKey generated by the address
  "ismine" : true|false,        (boolean) If the address is yours or not
  "iscoldstaking" : true|false, (boolean) If the address is a PIVX cold staking address\n"
  "iswatchonly" : true|false,   (boolean) If the address is watchonly
  "isscript" : true|false,      (boolean) If the key is a script
  "hex" : "hex",                (string, optional) The redeemscript for the P2SH address
  "pubkey" : "publickeyhex",    (string)  The hex value of the raw public key
  "iscompressed" : true|false,  (boolean) If the address is compressed
  "account" : "account"         (string)  The account associated with the address, "" is the default account
```

Reply:
I agree. But the current PR is not changing those lines.
Simply adds a new one there.
I think it would be better to address this styling nit in a separate PR (checking dis-aligned help outputs in ALL the rpc functions).